kythe/go/extractors/config/extractor.go

```diff
@@ -205,8 +205,21 @@ func ExtractRepo(ctx context.Context, repo Repo) error {
 		return fmt.Errorf("building docker image: %v\nCommand output %s", err, string(output))
 	}
 
+	targetRepoDir := repo.TempRepoDir
+	if targetRepoDir == "" {
+		targetRepoDir = DefaultRepoVolume
+	}
 	// run the extraction
-	output, err = exec.CommandContext(ctx, "docker", "run", "--rm", "-v", fmt.Sprintf("%s:%s", repoDir, DefaultRepoVolume), "-v", fmt.Sprintf("%s:%s", repo.OutputPath, DefaultOutputVolume), "-t", imageTag).CombinedOutput()
+	commandArgs := []string{"run", "--rm", "-v", fmt.Sprintf("%s:%s", repoDir, targetRepoDir), "-v", fmt.Sprintf("%s:%s", repo.OutputPath, repo.OutputPath), "-t", imageTag}
+	// Check and see if we're living inside a docker image.
```

Comment:
```
func inDockerImage(ctx context.Context) (string, bool)  {
   …
   return string(output), err == nil && len(output) != 0
}

if id, ok := inDockerImage(ctx); ok {
   args = append(args, "--voumes-from", id)
}
```

Note that `append` is variadic so you can append multiple individual arguments.

Reply:
ah of course it's variadic, dunno why I thought it wasn't.